### PR TITLE
Move some messages to trace

### PR DESCRIPTION
**Pull request by bazsi**
I find the latest messages clutter up debug messages a lot, so move some of the less interesting ones
to msg_trace(), now that we have that.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by szemere** [comment]
FYI: I am currently checking control flow, if there are any message "before/after" the changed ones which should be also changed as well.

File: lib/logproto/logproto-buffered-server.c [hunk 1]
```diff
@@ -667,8 +667,8 @@ log_proto_buffered_server_fetch_into_buffer(LogProtoBufferedServer *self)
   else if (rc == 0)
     {
       /* EOF read */
-      msg_verbose("EOF occurred while reading",
-                  evt_tag_int(EVT_TAG_FD, self->super.transport->fd));
+      msg_trace("EOF occurred while reading",
```
Comment (gaborznagy) [inline lib/logproto/logproto-buffered-server.c]: Would you change it in `logproto-framed-server.c` too, please?

**Comment by gaborznagy** [comment]
@bazsi what do you think about `Message was cloned` debug message?
I don't want to discuss every part I just came across it due to `default-network-drivers()`. :)

**Comment by bazsi** [comment]
I have now added the call sites requested in the comments, however I would not want to add more, so if you have any additions, please open another PR.

**Comment by MrAnno** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**